### ISSUE TITLE
Echo when switching to some internet radio stations

### ISSUE THREAD

**Issue by linknum23** [micro-nova/AmpliPi#400]
A discourse user reported an echo when changing internet radio stations. This likely means that the previous radio station continues to play. 
- [ ] Need a repeatable failing transition (url10 -> (url2)
- [ ] Investigate potential fixes in the run_vlc.py script used to play internet radio stations

**Comment by stamateviorel** [comment]
yes i wanted to add that from what i can see is mostly aac compressed streaming.

**Comment by linknum23** [comment]
Need to check into reported failures on this station: http://icecast-servers.vrtcdn.be/klara.aac specifically with announcements,

**Comment by linknum23** [comment]
This has been reported to still occur on some stations: http://icecast.vrtcdn.be/ra2vlb-high.mp3
